packages/desktop-client/src/components/settings/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { connect } from 'react-redux';
+
+import * as actions from 'loot-core/src/client/actions';
+import Platform from 'loot-core/src/client/platform';
+import { listen } from 'loot-core/src/platform/client/fetch';
+import { View, Text, Button } from 'loot-design/src/components/common';
+import { colors } from 'loot-design/src/style';
+import { mobileStyles } from 'loot-design/src/style';
+import { withThemeColor } from 'loot-design/src/util/withThemeColor';
+
+import useServerVersion from '../../hooks/useServerVersion';
+import { isMobile } from '../../util';
+import { Page } from '../Page';
+import EncryptionSettings from './Encryption';
+import Export from './Export';
+import FormatSettings from './Format';
+import GlobalSettings from './Global';
+import { ResetCache, ResetSync } from './Reset';
+import { Section, AdvancedToggle } from './UI';
+
+function About() {
+  const version = useServerVersion();
+
+  return (
+    <Section title="About" style={{ gap: 5 }}>
+      <Text>Client version: v{window.Actual.ACTUAL_VERSION}</Text>
+      <Text>Server version: {version}</Text>
+    </Section>
+  );
+}
+
+function AdvancedAbout({ prefs }) {
+  return (
+    <>
+      <Text>Budget ID: {prefs.id}</Text>
+      <Text style={{ color: colors.n6 }}>
+        Sync ID: {prefs.groupId || '(none)'}
+      </Text>
+    </>
+  );
+}
+
+function Settings({
+  loadPrefs,
+  savePrefs,
+  prefs,
+  globalPrefs,
+  pushModal,
+  resetSync,
+  closeBudget
+}) {
+  useEffect(() => {
+    let unlisten = listen('prefs-updated', () => {
+      loadPrefs();
+    });
+
+    loadPrefs();
```

Comment:
Nothing uses `userData` as far as I can see, so I don’t think it needs to be called.

Reply:
You're right. It seems to have been unnecessary since the very first commit even. Good catch!
 
First state: https://github.com/actualbudget/actual/blob/4d9fdfc590e500c340bbcb4f09f2ec8681b97407/packages/desktop-client/src/components/Settings.js#L48